test/e2e/settings/no_auth_locations.go

```diff
@@ -51,16 +51,15 @@ var _ = framework.IngressNginxDescribe("No Auth locations", func() {
 		Expect(s).NotTo(BeNil())
 		Expect(s.ObjectMeta).NotTo(BeNil())
 
-		updateConfigmap(setting, noAuthPath, f.KubeClientSet)
-
 		bi := buildBasicAuthIngressWithSecondPath(host, f.Namespace.Name, s.Name, noAuthPath)
 		ing, err := f.EnsureIngress(bi)
 		Expect(err).NotTo(HaveOccurred())
 		Expect(ing).NotTo(BeNil())
+
+		err = f.UpdateNginxConfigMapData(setting, noAuthPath)
```

Comment:
any specific reason you move this down below EnsureIngress?

Reply:
No reason here, could move it back up